fetch.bs

```diff
@@ -2118,6 +2118,50 @@ clearly stipulates that <a>connections</a> are keyed on
      WebSocket saner -->
 
 
+<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+
+<p>To
+<dfn lt="determine the HTTP cache partition|determining the HTTP cache partition">determine the HTTP cache partition</dfn>,
+given <var>request</var>, run these steps:
+
+<ol>
+ <li><p>Let <var>topLevelOrigin</var> be null.
+
+ <li>
+  <p>If <var>request</var>'s <a for=request>reserved client</a> is non-null, then:
+
+  <ol>
+   <li><p>Set <var>topLevelOrigin</var> to <var>request</var>'s <a for=request>reserved client</a>'s
+   <a for="environment">top-level origin</a>.
+
+   <li>
+    <p>If <var>topLevelOrigin</var> is null, then set <var>topLevelOrigin</var> to
+    <var>request</var>'s <a for=request>reserved client</a>'s
+    <a for="environment">top-level creation URL</a>'s <a for=url>origin</a>.
+
+    <p class=note>This happens for top-level navigations only.
+  </ol>
+
+ <li><p>Otherwise, if <var>request</var>'s <a for=request>client</a> is non-null, set
+ <var>topLevelOrigin</var> to <var>request</var>'s <a for=request>client</a>'s
+ <a for="environment">top-level origin</a>.
+
+ <li><p>Otherwise, return null.
+
+ <li><p>Assert: <var>topLevelOrigin</var> is an <a for=/>origin</a>.
+
+ <li><p>Let <var>topLevelSite</var> be the result of <a lt="obtain a site">obtaining a site</a>,
+ given <var>topLevelOrigin</var>.
+
+ <li>
+  <p>Return the HTTP cache associated with <var>topLevelSite</var> and, possibly, a second key.
+  [[!HTTP-CACHING]]
+
+  <p class=XXX>The second key is intentionally a little vague as the finer points are still
+  evolving. See <a href=https://github.com/whatwg/fetch/issues/904>issue #904</a>.
```

Comment:
Would this be the best link for this or should we add either the privacycg storage partitioning issue link or the https://github.com/shivanigithub/http-cache-partitioning/issues/2?

Reply:
I guess that issue and https://privacycg.github.io/storage-partitioning/ probably do a better job of capturing things. What do you think about porting that issue into this repository as a new issue, link that from here, and make this PR close 904?

Reply:
Do you mean the shivanigithub/http-cache-partitioning#2 issue to be ported to a new issue in https://privacycg.github.io/storage-partitioning/ ? I think this plan sounds good.

Reply:
I meant here in the Fetch repository, but that seems fine too. I'll get to this on Monday.

Reply:
Yeah fetch repository seems fine too.